tailwind.config.mjs

```diff
@@ -1,10 +1,14 @@
 import tailwindConfig from '@hypothesis/frontend-shared/lib/tailwind.preset.js';
 
+const successGreen =
+  tailwindConfig.theme?.extend?.colors?.green?.success ?? '#00a36d';
```

Comment:
So much optional chaining!

Reply:
YES! NASTY! This is temporary until the next `frontend-shared` release, after which I will remove this and any other `theme` entries that are available from the upstream `preset`.